src/style/common/like.less

```diff
@@ -54,7 +54,7 @@
 
 .like-button-liked {
   > span {
-    color: var(--gray-70) !important;
+    color: @red-500 !important;
```

Comment:
Didn't Wolfgang specify dark-red-500?

Reply:
[Figma](https://www.figma.com/file/OrP9jAJdfttfvsm3nxnihP/Accessibility-Quick-Wins-(Web%2FDesktop)?node-id=924%3A14200), click on the heart icon and you can see the colour mentioned for the heart icon is
Light UI/Red 500 #C20013

If thats wrong then the figma needs to be updated.

Reply:
gotcha, I imagine Wolfgang is using Dark Mode and is trying to say the heart color should be different depending on the theme. I agree it's a mess, there's no Dark Mode exemple in Figma and I interpreted Wolfgang comment as "heart is @dark-red-500 all the time".

You do need to change the title though, you can't merge a PR without semantic commit title